examples/weather/fcn_afno/conf/config.yaml

```diff
@@ -20,12 +20,26 @@ hydra:
   run:
     dir: ./outputs/
 
+train_dir: ./dataset/train
```

Comment:
Can we keep the defaults to `/data/*`? This example has been integrated with a few workflows internally, like perf testing, qa, etc. and I think this change will break that

Reply:
Great point - I forgot about our internal QA workflows!